green/environment-dependent-configuration-parameters.md

```diff
@@ -0,0 +1,25 @@
+# Environment-dependent configuration parameters
+
+Provide all environment-dependent configuration as parameters. Avoid hard-coded environment-dependent data (e.g., credentials, paths) in application and infrastructure source code.
+
+## Benefits
+
+- No leakage of confidential data through compromised source code.
+- Changing configurations of your application does not require source code changes and redeployment.
+- Credentials are not located in the version control system repositories.
+- The tested source code is the one which is deployed. No environment specific changes are required right before production deployment. Therefore, bugs introduced by these changes are prevented.
+
+## Assessment
+
+- Show your configuration files for all your different environments like dev, test and production
+  - URL specification for different environments
+  - Configuration parameters for backend / database systems
+
+## Related Activities
+
+- [Same Artifact for Environments](../orange/same-artifact-for-environments.md)
```

Comment:
I think I would like it to mark activities which are not of this belt, e.g., "Orange Belt: Same Artificat ..."